ai2thor/controller.py

```diff
@@ -392,6 +392,7 @@ def __init__(
             local_build = True
 
         self.last_event = None
+        self.current_scene_name = None
```

Comment:
Would you be opposed to naming this self.scene instead of self.current_scene_name?  Since there is only ever one scene loaded it won't conflict with anything and then the member variable will match the parameter to the constructor which makes the property a bit easier to discover by the end user.

Reply:
Done